Header header

# 
float64 pressureDiff1
float64 pressureDiff2

bool fan1				# 
bool fan2				# 
	
bool valve				# 
bool valve_helium		# 